File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/WordObject.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class WordObject {
  @BeanProperty var id: Long = 0L
  @BeanProperty var word: String = _
  @BeanProperty var vulgar: String = _
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class WordObject {\n")
    sb.append("  id: ").append(id).append("\n")
    sb.append("  word: ").append(word).append("\n")
    sb.append("  vulgar: ").append(vulgar).append("\n")
    sb.append("}\n")
    sb.toString
  }
}

File: samples/wordnik-api-java/src/main/java/com/wordnik/client/model/WordObject.java
package com.wordnik.client.model;

public class WordObject {
  private long id = 0L;
  private String word = null;
  private String vulgar = null;
  public long getId() {
    return id;
  }
  public void setId(long id) {
    this.id = id;
  }

  public String getWord() {
    return word;
  }
  public void setWord(String word) {
    this.word = word;
  }

  public String getVulgar() {
    return vulgar;
  }
  public void setVulgar(String vulgar) {
    this.vulgar = vulgar;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class WordObject {\n");
    sb.append("  id: ").append(id).append("\n");
    sb.append("  word: ").append(word).append("\n");
    sb.append("  vulgar: ").append(vulgar).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
